GitExtUtils/GitUI/TaskExtensions.cs

```diff
@@ -6,195 +6,195 @@
 
 namespace GitUI
 {
-    public static class TaskExtensions
```

Comment:
This file will be removed.
Appears the class is obsolete with the new Microsoft.VisualStudio.Threading